### PR TITLE
Patch build docs and add a warning to womtool doc about graph failing on 1.0

### PR DESCRIPTION
**Pull request by vdauwera**
- Build docs were vague on location of womtool JAR
- Womtool graph fails on WDL 1.0 so we should warn people not to waste their time until it's fixed

### CODE REVIEW

File: docs/WOMtool.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The following is the toolchain used for development of womtool.  Other versions
 
 ## Building
 
-`sbt assembly` will build a runnable JAR in `target/scala-2.11/`
+`sbt assembly` will build a runnable JAR in `womtool/target/scala-2.11/`
```
Comment (cjllanwarne) [inline docs/WOMtool.md]: Not `2.12`?

File: docs/developers/Building.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ $ sbt assembly
 NOTE: This command will run for a long time the first time.  
 NOTE: Compiling will not succeed on directories encrypted with ecryptfs (ubuntu encrypted home dirs for example), due to long file paths.
 
-`sbt assembly` will build the runnable Cromwell JAR in `server/target/scala-2.12/` with a name like `cromwell-<VERSION>.jar`.
+`sbt assembly` will build the runnable Cromwell JAR in `server/target/scala-2.12/` with a name like `cromwell-<VERSION>.jar`. It will also build a runnable Womtool JAR in `womtool/target/scala-2.12/` with a name like `cromwell-<VERSION>.jar`.
```
Comment (cjllanwarne) [inline docs/developers/Building.md]: with a name like `womtool-<VERSION>.jar`?

File: docs/WOMtool.md (hunk 1 above)
Comment (vdauwera) [reply docs/WOMtool.md]: Oh sure we can update to whatever is the latest now.

File: docs/developers/Building.md (hunk 2 above)
Comment (vdauwera) [reply docs/developers/Building.md]: Doh, yes

**Comment by vdauwera** [comment]
@cjllanwarne Thanks for picking up those two errors, I fixed them. Should be all set now.

**Pull request merged**